### ISSUE TITLE
[model] @transient annotation on property

### ISSUE THREAD

**Issue by jcheron** [phpMv/ubiquity#113]
### Steps
Trying to add a `@transient` annotation on a property in a model class so that the property is not serializable. 
```php
/**
* @transient
*/
private bool $new = false;
```
### Expected Result
- No error
- The property `$new` is ignored in DAO operations
### Actual Result
Models Cache generation display an error
```bash
Ubiquity init-cache -t=models
```
>"Message : Annotation type 'Ubiquity\annotations\TransientAnnotation' cannot be applied to a property"

### Versions
- Ubiquity framework 2.3.5
- php *
- OS *

@JamesMahy

**Comment by jcheron** [comment]
@JamesMahy

Logically, this is solved in [ubiquity-dev 0.0.7 release](https://github.com/phpMv/ubiquity-dev/releases/tag/0.0.7)

Can you confirm?

**Comment by jcheron** [comment]
Fixed in [2.3.6 release](https://github.com/phpMv/ubiquity/releases/tag/2.3.6)
